lib/client/db/postgres/servicefile.go

```diff
@@ -167,10 +97,14 @@ func (s *ServiceFile) Upsert(profile ConnectProfile) error {
 	if profile.Database != "" {
 		section.NewKey("dbname", profile.Database)
 	}
-	section.NewKey("sslmode", profile.SSLMode)
-	section.NewKey("sslrootcert", profile.SSLRootCert)
-	section.NewKey("sslcert", profile.SSLCert)
-	section.NewKey("sslkey", profile.SSLKey)
+	if profile.Insecure {
+		section.NewKey("sslmode", SSLModeVerifyCA)
```

Comment:
According to [documentation](https://www.postgresql.org/docs/9.3/libpq-connect.html#LIBPQ-CONNECT-SSLMODE), there are 6 SSL modes and the 3 `require`, `verify-ca` and `verify-full` will error if Postgres was not compiled with SSL support - is this a concern in this case? Also looks like `disable` or even `allow` would make a better candidate for insecure mode than `verify-ca`.

Reply:
It's not a concern because we require TLS for the database access. I'm using verify-ca for the same reason, all our routing/authentication it based on client certificates. Also, this is configuring client to talk to Teleport proxy, not Postgres.